### PR TITLE
add simple conv2d problem

### PR DESCRIPTION
**Pull request by drogovozDP**

### CODE REVIEW

File: Problems/40_simple_conv2d_layer/learn.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Convolutional layer is widely used in Computer Vision tasks. Here is crucial parameters:
+
+<h3>Parameters:</h3>
+<ol>
+    <li>
+        <b>input_matrix:</b> is a 2D NumPy array representing the input data, such as an image. Each element in this array corresponds to a pixel or a feature value in the input space. The dimensions of the input matrix are typically denoted as \[\text{(height, width)}\].
+    </li>
+    <li>
+       <b>kernel:</b> is another 2D NumPy array representing the convolutional filter. The kernel is smaller than the input matrix and slides over it to perform the convolution operation. Each element in the kernel represents a weight that modifies the input as it is convolved over it. The kernel size is denoted as `(kernel_height, kernel_width)`.
```
Comment (Open-Deep-ML) [inline Problems/40_simple_conv2d_layer/learn.html:13]: `(kernel_height, kernel_width)` renders a little strange a fix could be `(kernelHeight, kernelWidth)` seems like the "_" between then makes it render with the h and as a subscript

File: Problems/40_simple_conv2d_layer/learn.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<h2>Simple Convolutional 2D Layer</h2>
+
+\[\text{output}_\text{height}\]
```
Comment (Open-Deep-ML) [inline Problems/40_simple_conv2d_layer/learn.html:3]: not sure if this line is needed

**Review by Open-Deep-ML — COMMENTED** [review]
This is a great question! I have a few suggestions regarding the formatting of learn.html. Also, could you please update this to be problem 41? I'm still in the process of gathering questions from deep-ml, and I'd like to keep everything consistent. For future reference, could you create an issue labeled 'New Problem' before starting work? This will help ensure that others aren't working on the same thing

File: Problems/40_simple_conv2d_layer/learn.html (hunk 2 above)
Comment (drogovozDP) [reply Problems/40_simple_conv2d_layer/learn.html]: Removed

File: Problems/40_simple_conv2d_layer/learn.html (hunk 1 above)
Comment (drogovozDP) [reply Problems/40_simple_conv2d_layer/learn.html]: Fixed. Now it `kernel_height` and `kernel_width` render correctly

**Review by Open-Deep-ML — APPROVED** [review]

**Pull request merged**